docs/shared/features/remote-cache.md

```diff
@@ -1,87 +1,71 @@
 # Use Remote Caching (Nx Replay)
 
-By default Nx [caches task computations locally](/features/cache-task-results). However, to benefit from the cache across your team and in particular on CI, the computation cache can also be distributed across multiple machines.
-
-The **Nx Replay** feature of Nx Cloud is a fast, secure and zero-config implementation of remote caching.
+Repeatedly rebuilding and retesting the same code is costly — not just in terms of wasted resources, but also in terms of developer time. To solve this, Nx includes a sophisticated computation caching system that ensures **code is never rebuilt twice**, not just saving you time but also resources.
 
 ![Diagram showing Teika sharing his cache with CI, Kimiko and James](/shared/images/dte/distributed-caching.svg)
 
-In this diagram, Teika runs the build once on his machine, then CI, Kimiko and James can use the cached artifact from Teika instead of re-executing the same work.
```

Comment:
I don't think the diagram is clear without some explanatory text.  No one knows what the arrows mean until we explain them.